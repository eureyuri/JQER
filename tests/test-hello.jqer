int def main():
  print("hello");
  print(true);
  print(true and false);
  print(true or false);
end
