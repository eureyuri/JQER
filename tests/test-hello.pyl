int def main():
  prints("hello");
end
