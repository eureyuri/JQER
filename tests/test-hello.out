hello
1
0
1
